docs/getting-started/quick-start.md

```diff
@@ -2,14 +2,185 @@
+PostgreSQL can be setup using the [Crunchy PostgreSQL operator from
+OperatorHub.io][crunchy].
+
+The installation of the operator doesn't create a database instance for
+connection.
+
+1. To create a database instance, you need to create custom resource
+(`Pgcluster`) and that will trigger the operator reconciliation.  For
+convenience, run this command to create `Pgcluster` custom resource:
+
+```bash
+kubectl apply -f https://gist.githubusercontent.com/baijum/b99cd8e542868a00b2b5efc2e1b7dc10/raw/11e790fb1d23aa4d3ee03c260169a08b36fb25bc/pgcluster.yaml
+```
+
+2. Ensure all the pods in `my-postgresql` is running (it will take few minutes):
+
+```bash
+kubectl get pod -n my-postgresql
+```
+
+You should see output something like this:
+
+```
+NAME                                          READY   STATUS    RESTARTS   AGE
+backrest-backup-hippo-9gtqf                   1/1     Running   0          13s
+hippo-597dd64d66-4ztww                        1/1     Running   0          3m33s
+hippo-backrest-shared-repo-66ddc6cf77-sjgqp   1/1     Running   0          4m27s
+```
+
+The database will be empty by default, it requires the schema and sample data to
+work with the application.
+
+3. You can initialize the database with the schema and sample data using this
+command:
+
+```bash
+bash <(curl -s https://gist.githubusercontent.com/baijum/b99cd8e542868a00b2b5efc2e1b7dc10/raw/11e790fb1d23aa4d3ee03c260169a08b36fb25bc/init-database.sh)>
+```
+
+Now the database is ready to connect from application.  The next section
+explains how to configure application:
+
+## Deploying an application
+
+1. Deploy the `spring-petclinic-rest` app with this `Deployment` configuration:
+
+```bash
+kubectl apply -f https://gist.githubusercontent.com/baijum/b99cd8e542868a00b2b5efc2e1b7dc10/raw/11e790fb1d23aa4d3ee03c260169a08b36fb25bc/app-deployment.yaml
+```
+
+2. Port forward the application port and try to access it from your local system
+
+```
+kubectl port-forward --address 0.0.0.0 svc/spring-petclinic-rest 9966:80 -n my-postgresql
+```
+
+3. open [http://localhost:9966/petclinic](http://localhost:9966/petclinic)
+
+You should see a [Swagger UI][swagger] where you can play with the API.
+
+Since the binding is not present in the application, you cannot see be any
+values in results.
+
+If you try to access list of all pets, you can see an error like this:
+
+```
+curl -X GET "http://localhost:9966/petclinic/api/pets" -H "accept: application/json"
+
+{"className":"org.springframework.transaction.CannotCreateTransactionException","exMessage":"Could
+not open JPA EntityManager for transaction; nested exception is
+org.hibernate.exception.JDBCConnectionException: Unable to acquire JDBC
+Connection"}
+```
+
+In the next section, you will see how to fix it.
+
+## Connecting the application to a backing service
+
+The application was not working as the bindings were not present in the app.
+
+1. Create the ServiceBinding custom resource to inject the bindings:
+
+```yaml
```

Comment:
let's explain this yaml:
- What are the two main sections
- How do we map the service/application in the SB resource
- why do we have 2 services
one who is not familiar with it, will not understand otherwise.

Reply:
Done. 1c27e60b4ef80faf802e6913a35b59dcc746f62a